rpyc/core/protocol.py

```diff
@@ -560,6 +560,15 @@ def _handle_setattr(self, oid, name, value):
         return self._access_attr(oid, name, (value,), "_rpyc_setattr", "allow_setattr", setattr)
     def _handle_callattr(self, oid, name, args, kwargs):
         return self._handle_getattr(oid, name)(*args, **dict(kwargs))
+    def _handle_ctxexit(self, oid, exc):
+        if exc:
```

Comment:
Indeed it's to provide a valid traceback object. My only fear is CMs that are written naively, like:
```
def __exit__(self, *args):
    if args[0]:
        log_traceback(args[2])
```
In other words, expecting there to always be a traceback object if an exception object was passed.